oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -536,8 +536,8 @@ impl wasmi::Externals for WasmInterface {
             WAIT_ON_CHANNELS => {
                 map_host_errors(self.wait_on_channels(args.nth_checked(0)?, args.nth_checked(1)?))
             }
-            WASI_STUB => panic!("Attempt to invoke unimplemented WASI function"),
-            _ => panic!("Unimplemented function at {}", index),
+            WASI_STUB => panic!("Attempt to invoke unimplemented WASI function"), // LCOV_EXCL_LINE
```

Comment:
Are we expected to annotate code in this way going forward? If so, all panics, or just some?

Reply:
Yeah, that would be a pain &ndash; I've dropped the manual exclusion, and set `panic!` and `unreachable!` to count as unhittable.